sway/server.c

```diff
@@ -73,12 +73,17 @@ static void handle_drm_lease_request(struct wl_listener *listener, void *data) {
 bool server_init(struct sway_server *server) {
 	sway_log(SWAY_DEBUG, "Initializing Wayland server");
 
-	struct wlr_renderer *renderer = wlr_backend_get_renderer(server->backend);
-	assert(renderer);
+	server->renderer = wlr_renderer_autocreate(server->backend);
+	assert(server->renderer);
```

Comment:
Before this wlroots breaking change, `wlr_backend_autocreate` was responsible for creating the renderer. That's no longer the case. That means `wlr_renderer_autocreate` will return NULL if the renderer couldn't be created.

Can we add proper error handling instead of assertions for `wlr_renderer_autocreate` and `wlr_allocator_autocreate`?